org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/commons/config/ConfigurationItemEdit.java

```diff
@@ -23,6 +23,10 @@ public class ConfigurationItemEdit extends ConfigurationItem {
 	private ConfigurationItemEditType editType;
 	private ConfigurationItemValueKind valueKind;
 
+	public ConfigurationItemEdit () {
+		// Need explicit no-arg constructor for native image
```

Comment:
This is needed for Gson, not graalvm native image

Reply:
I'll update.